src/lib/getPriorityIndex.ts

```diff
@@ -223,37 +233,27 @@ const priorityGroups: readonly PriorityGroup[] = [
       "borderLeftWidth",
       "borderLeftStyle",
       "borderLeftColor",
-      "borderStart",
-      "borderStartWidth",
-      "borderStartStyle",
-      "borderStartColor",
-      "borderEnd",
-      "borderEndWidth",
-      "borderEndStyle",
-      "borderEndColor",
-      "borderX",
-      "borderY",
     ],
     priority: stylePropsPriority["Border"],
   },
   {
     name: "Border Radius",
     keys: [
       "borderRadius",
-      "borderTopLeftRadius",
-      "borderTopStartRadius",
-      "borderTopRightRadius",
-      "borderTopEndRadius",
-      "borderBottomRightRadius",
-      "borderBottomEndRadius",
-      "borderBottomLeftRadius",
-      "borderBottomStartRadius",
+      "borderStartRadius",
+      "borderEndRadius",
       "borderTopRadius",
       "borderRightRadius",
-      "borderEndRadius",
       "borderBottomRadius",
       "borderLeftRadius",
-      "borderStartRadius",
+      "borderTopStartRadius",
+      "borderTopEndRadius",
+      "borderTopRightRadius",
+      "borderTopLeftRadius",
+      "borderBottomStartRadius",
+      "borderBottomEndRadius",
+      "borderBottomLeftRadius",
+      "borderBottomRightRadius",
```

Comment:
I think these should appear before `borderStartRadius` because `borderTopStartRadius` is more wide property than `borderStartRadius`, like `borderX` is more wide than `borderStart`.

Reply:
BorderStart is 2 corners, borderTopStart is just 1 corner

Reply:
Oh sorry, i i was misunderstanding. Please forget about it.